tower-reconnect/src/lib.rs

```diff
@@ -4,11 +4,15 @@ extern crate log;
 extern crate tower_service;
 extern crate tower_util;
 
+pub mod future;
+
+use crate::future::ResponseFuture;
```

Comment:
I think this is illegal until you turn on 2018 edition.

Reply:
doesn't seem like it!

Reply:
2018 isn't enabled in the crate, CI is passing. Is there something else to check?